### DISCUSSION TITLE
Unbound with nspawn

### DISCUSSION THREAD

**Discussion by jasonhollis** [unifi-utilities/unifi-common#611] [Q&A]
I have the nsspawn core container setup and running fine.  I have used simple networking e.g. it looks like my custom-delian is connected to all of the interfaces on the DMP-SE.  hostname -I gives me: 203.XXX.XXX.XXX 192.168.90.1 192.168.90.2 192.168.3.1 2404:XXX:XXX This make sense to me in these are the external an internal addresses of the UDMP-SE the 3.1 address I guess is a new network that was spawned for the containers?  I'm not sure why I would want to run a seperate VLAN and address space for the containers on the UDMP-SE or MAC-VLAN.  All I really want to do is run Unbound via nsspawn on the UDMP-SE.  I don't want or need pi-hole because I'm already running NextDNS.  I just want Unbound running locally pointed upstream to NextDNS.  I'm pretty bogged down at this point trying to figure out how to make this happen?

**Comment by m4n63** [comment]
I have not used unbound, but cant you just install it in the container?
ie login to the container:
``` machinectl shell debian-custom ``` (or whatever name you used)
and then from within the container
```
apt-get update
apt-get install unbound
```
Or am I misunderstanding your question?
